### PR TITLE
[#1090] Oracle 5.11 (gateway to CentOS5.11, RHEL5.11)

### PR DESCRIPTION
**Pull request by theopolis**
Very WIP for Legacy Linux OSes

### CODE REVIEW

**Comment by theopolis** [comment]
ok to test

File: tools/lib.sh [hunk 1]
```diff
@@ -7,18 +7,26 @@
...
 
+ORACLE_RELEASE=/etc/oracle-release
 SYSTEM_RELEASE=/etc/system-release
 UBUNTU_RELEASE=/etc/lsb-release
 
 function platform() {
   local  __out=$1
-  if [[ -n `grep -o "CentOS" $SYSTEM_RELEASE 2>/dev/null` ]]; then
+  if [[ -f "$ORACLE_RELEASE" ]]; then
+    FAMILY="redhat"
```
Comment (theopolis) [inline tools/lib.sh]: @javuto so if I declare these `$FAMILY` variables within a call to `platform $OS` I should be ok?

**Comment by theopolis** [comment]
ok to test

File: tools/lib.sh (hunk 1 above)
Comment (javuto) [inline tools/lib.sh]: Those will be valid during the execution of the script. If you want to persist those values in the shell, then add ```export``` but I think you are good here

**Comment by theopolis** [comment]
ok to test

**Comment by theopolis** [comment]
ok to test

**Pull request merged**